cms/templates/js/no-content-groups.underscore

```diff
@@ -0,0 +1,3 @@
+<div class="no-group-configurations-content">
```

Comment:
I think I prefer having a single template for the view which dynamically decides whether to show the 'no content' version of itself. Then the view just has to re-render itself and the view will update. Having a separate template isn't terrible for two cases, but I think it could get worse if other cases come up.